install/installer/docs/workspace-ssh-access.md

```diff
@@ -0,0 +1,26 @@
+# Installing Gitpod with SSH access to workspaces with the Gitpod Installer
+**_Needed by some desktop IDEs to connect to a workspace._**
+
+> **IMPORTANT:** This uses port 22 on your Kubernetes nodes. When enabled, this will prevent login to the cluster via SSH. If you wish to maintain SSH access to your cluster, please configure another SSH port on your nodes.
+
+To enable the SSH gateway you need to generate a host key, need to add it as secret to your Kubernetes cluster, and need to configure the Gitpod Installer to use this secret.
+
+You can use `ssh-keygen` to generate a host key like this:
+
+```shell
+$ ssh-keygen -t rsa -q -N "" -f host.key
```

Comment:
Might be a nit, but I never like bash that starts with `$` as you can't just copy/paste